Experimental/MultiplayerUseCases/Assets/UseCaseSamples/NetvarVsRpc/Scripts/ColorManager.cs

```diff
@@ -65,23 +65,23 @@ void Update()
 
         void OnClientRequestColorChange()
         {
-            OnServerChangeColorServerRpc();
+            ServerChangeColorRpc();
         }
 
-        [ServerRpc(RequireOwnership = false)] //note: please refer to RPCs documentation to learn more about the pros and cons of the RequireOwnership parameter
-        void OnServerChangeColorServerRpc()
+        [Rpc(SendTo.Server, RequireOwnership = false)] //note: please refer to Rpcs documentation to learn more about the pros and cons of the RequireOwnership parameter
+        void ServerChangeColorRpc()
```

Comment:
In order to maintain consistency with NGO's naming (and with the gaming industry's best practice), methods in NetworkBehaviours that are executed on Client or Server should start with "OnClient" or "OnServer" for clarity.

@Elfi0Kuhndorf Since you're going through renaming RPCs, could you rename according to that logic?

Reply:
That is another one where we should align on, because in the other samples we also have it without the "On" see here https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/172/files/ac62665febabdd5e25ba3547e47656a6122f55d8#diff-6b9abec6c4023027ebb424041550534e899d05dcd4a7384cfe31af5ffad70a1e

Reply:
For the purposes of getting this PR through, let's stick with Netcode for GameObject's current Rpc abbreviation and finish these conversations offline.

Reply:
I agree with this. I will leave the Rpc naming as it is currently and later once we decided on abbreviations offline we can adjust our projects accordingly.